src/app/examples/examples.module.ts

```diff
@@ -24,12 +24,19 @@ import { BooksEffects } from './crud/books.effects';
 import { FormComponent } from './form/components/form.component';
 import { FormEffects } from './form/form.effects';
 import { AuthenticatedComponent } from './authenticated/authenticated.component';
+import {
+  StoreRouterConnectingModule,
+  RouterStateSerializer
+} from '@ngrx/router-store';
 
 @NgModule({
   imports: [
     SharedModule,
     ExamplesRoutingModule,
     StoreModule.forFeature(FEATURE_NAME, reducers),
+    StoreRouterConnectingModule.forRoot({
```

Comment:
Move this to `core.module`, the default `stateKey` value is router so we can change this to:

```
StoreRouterConnectingModule.forRoot()
```

Or is it more clear if we would do it like yo had?

Reply:
It was indeed a first try to learn that can be cleaned using de default value.